src/pages/apis/device/covid19serology/_infographic_definitions.json

```diff
@@ -0,0 +1,355 @@
+{
+  "globalDefs": {
+    "startYear": 2009,
+    "api": "/drug/event",
```

Comment:
@beardedfinch

Reply:
We didn't end up using this time of infographic. I'll delete the file.